TW5_parse_top_stats_tools.py

```diff
@@ -1673,19 +1723,55 @@ def collect_stat_data(args, config, log, anonymize=False):
 				if player.stats_per_fight[fight_number][stat] > 0:
 					# buff are generation squad values, using total over time
 					if stat in config.buffs_stacking_duration and stat != 'iol':
-						#value is generated boon time on all squad players / fight duration / (players-1)" in percent, we want generated boon time on all squad players / (players-1)
-						fight.total_stats[stat] += round(player.stats_per_fight[fight_number][stat]/100.*fight.duration, 4)
-						player.total_stats[stat] += round(player.stats_per_fight[fight_number][stat]/100.*fight.duration, 4)
+						#value is generated boon time on all squad players / fight duration / (players-1)" in percent, we want generated boon time on all squad players
+						fight.total_stats[stat] += round(player.stats_per_fight[fight_number][stat]/100.*fight.duration*(fight.allies - 1), 4)
+						player.total_stats[stat] += round(player.stats_per_fight[fight_number][stat]/100.*fight.duration*(fight.allies - 1), 4)
+
+						group_gen = get_stat_from_player_json(player_data, players_running_healing_addon, stat, config, False, BuffGenerationType.GROUP)
+						player.total_stats_group[stat] += round(group_gen/100.*fight.duration*(num_party_members - 1), 4)
+
+						in_combat_gen = get_stat_from_player_json(player_data, players_running_healing_addon, stat, config, True, BuffGenerationType.SQUAD)
+						player.total_stats_in_combat[stat] += round(in_combat_gen/100.*player.stats_per_fight[fight_number]['time_in_combat']*(fight.allies - 1), 4)

```

Comment:
Produces values > player.total_stats[stat] under some circumstances.  Not sure what they are.
Find it more in the following stats:

- regen
- alacrity
- superspeed